__init__.py

```
@@ -2340,3 +2518,238 @@ async def async_schedule_appliance(call) -> None:
+            recent_logs = list(all_logs)[-20:]
+
+        # Read SEM integration version (best effort)
+        sem_version = "unknown"
+        try:
+            import os as _os
+            import json as _json_v
+            manifest = _os.path.join(_os.path.dirname(__file__), "manifest.json")
+            with open(manifest) as _f:
+                sem_version = _json_v.load(_f).get("version", "unknown")
+        except Exception:  # noqa: BLE001
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)